src/libgit2.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) the libgit2 contributors. All rights reserved.
+ *
+ * This file is part of libgit2, distributed under the GNU GPL v2 with
+ * a Linking Exception. For full terms see the included COPYING file.
+ */
+#ifndef INCLUDE_libgit2_h__
+#define INCLUDE_libgit2_h__
+
+extern int git_libgit2_settings_global_init(void);
```

Comment:
This declaration shouldn't be needed at all, right? We're making direct use of it in `libgit2.c`.

Reply:
Fixed.